tracer/dependabot/Datadog.Dependabot.Integrations.csproj

```diff
@@ -195,28 +195,28 @@
 
     <!-- Integration: Microsoft.Data.Sqlite -->
     <!--    Assembly: Microsoft.Data.Sqlite -->
-    <!-- Latest package https://www.nuget.org/packages/Microsoft.Data.Sqlite/9.0.4 -->
-    <PackageReference Include="Microsoft.Data.Sqlite" Version="9.0.4" />
+    <!-- Latest package https://www.nuget.org/packages/Microsoft.Data.Sqlite/9.0.5 -->
+    <PackageReference Include="Microsoft.Data.Sqlite" Version="9.0.5" />
 
     <!-- Integration: Microsoft.Extensions.Identity.Core -->
     <!--    Assembly: Microsoft.Extensions.Identity.Core -->
-    <!-- Latest package https://www.nuget.org/packages/Microsoft.Extensions.Identity.Core/9.0.4 -->
-    <PackageReference Include="Microsoft.Extensions.Identity.Core" Version="9.0.4" />
+    <!-- Latest package https://www.nuget.org/packages/Microsoft.Extensions.Identity.Core/9.0.5 -->
+    <PackageReference Include="Microsoft.Extensions.Identity.Core" Version="9.0.5" />
 
     <!-- Integration: Microsoft.Extensions.Logging -->
     <!--    Assembly: Microsoft.Extensions.Logging -->
-    <!-- Latest package https://www.nuget.org/packages/Microsoft.Extensions.Logging/9.0.4 -->
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="9.0.4" />
+    <!-- Latest package https://www.nuget.org/packages/Microsoft.Extensions.Logging/9.0.5 -->
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="9.0.5" />
 
     <!-- Integration: Microsoft.Extensions.Logging.Abstractions -->
     <!--    Assembly: Microsoft.Extensions.Logging.Abstractions -->
-    <!-- Latest package https://www.nuget.org/packages/Microsoft.Extensions.Logging.Abstractions/9.0.4 -->
-    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="9.0.4" />
+    <!-- Latest package https://www.nuget.org/packages/Microsoft.Extensions.Logging.Abstractions/9.0.5 -->
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="9.0.5" />
 
     <!-- Integration: Microsoft.Extensions.Telemetry -->
     <!--    Assembly: Microsoft.Extensions.Telemetry -->
-    <!-- Latest package https://www.nuget.org/packages/Microsoft.Extensions.Telemetry/9.4.0 -->
-    <PackageReference Include="Microsoft.Extensions.Telemetry" Version="9.4.0" />
+    <!-- Latest package https://www.nuget.org/packages/Microsoft.Extensions.Telemetry/9.5.0 -->
+    <PackageReference Include="Microsoft.Extensions.Telemetry" Version="9.5.0" />
```

Comment:
I think this has broken the current integration based on failing integration tests? Needs looking into

Reply:
New params in constructor: https://github.com/dotnet/extensions/pull/5635/files#diff-fe41c81f934acba2a6833c4ea862466b39858fc5e559ccf82263d57ce920a597